engine/src/main/java/org/terasology/engine/persistence/typeHandling/TypeHandlerLibraryImpl.java

```diff
@@ -113,6 +115,7 @@ private static void populateWithDefaultHandlers(TypeHandlerLibrary serialization
         serializationLibrary.addTypeHandlerFactory(new AssetTypeHandlerFactory());
 
         serializationLibrary.addTypeHandler(Name.class, new NameTypeHandler());
+        serializationLibrary.addTypeHandler(SimpleUri.class, new SimpleUriTypeHandler());
```

Comment:
We'll want to add BlockUri's handler here too, right?

Reply:
good catch, thank you! will add that later today 👍